CMakeLists.txt

```diff
@@ -736,6 +736,11 @@ configure_file(${cdat_CMAKE_SOURCE_DIR}/cdat_modules_extra/runtest.in
   @ONLY
 )
 
+configure_file(${cdat_CMAKE_SOURCE_DIR}/link_python.sh.in
+  ${cdat_BINARY_DIR}/link_python.sh
+  @ONLY
+)
```

Comment:
I'd rather not use configure_file as it would add to the already huge project configuration time.

You can do something like this at install time. 
```
# try to create a symlink
execute_process(
  COMMAND "${CMAKE_COMMAND}" "-E" "create_symlink" "python2.7" "python"
  WORKING_DIRECTORY "${CMAKE_INSTALL_PREFIX}/bin"
  RESULT_VARIABLE rc
  OUTPUT_VARIABLE output
  ERROR_VARIABLE error
)
```